### PR TITLE
Update information at documentation main page #2360

### PR DESCRIPTION
**Pull request by FanJups**
Update information at documentation main page
closes: https://github.com/mercedes-benz/sechub/issues/2360

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -45,10 +45,12 @@ <h1 id="in-a-nutshell">In a nutshell</h1>
 
         <p>SecHub represents a mechanism to integrate diverse security products like</p>
         <ul>
-          <li>static code analysis tools</li>
-          <li>web scanners</li>
-          <li>infrastructure scanners</li>
-          <li>… maybe more in future</li>
+          <li>SAST (static application security tools)</li>
```
Comment (de-jcup) [inline docs/index.html:48]:
```suggestion
          <li>SAST (Static Application Security Testing)</li>
```

File: docs/index.html [hunk 2]
```diff
@@ -45,10 +45,12 @@ <h1 id="in-a-nutshell">In a nutshell</h1>
...
+          <li>DAST (dynamic application security tools)</li>
```
Comment (de-jcup) [inline docs/index.html:49]:
```suggestion
          <li>DAST (Dynamic Application Security Testing)</li>
```

**Review by de-jcup — CHANGES_REQUESTED** [review]
Thank you for your contribution but I still want some small changes, see my comments.

File: docs/index.html (hunk 1 above)
Comment (FanJups) [reply docs/index.html]: ok, it's done

File: docs/index.html (hunk 2 above)
Comment (FanJups) [reply docs/index.html]: ok, it's done

File: docs/index.html (hunk 1 above)
Comment (de-jcup) [reply docs/index.html]: @FanJups : Sorry, but please remove the gartner links ... It's not clear if the company allows deep links inside - I just added the links to give you the information.

But I think your idea about adding links for DAST and AST is very good.
https://en.wikipedia.org/wiki/Static_application_security_testing would be a good aceptable link for SAST

**Comment by FanJups** [comment]
> Please replace the Gartner links for `SAST` and `DAST` with the wikipedia pendants:
> 
> * https://en.wikipedia.org/wiki/Static_application_security_testing
> * https://en.wikipedia.org/wiki/Dynamic_application_security_testing
> 
> as mentioned in my comments.

ok its done.

**Review by de-jcup — APPROVED** [review]
Looks good to me :+1: 

Thank you for your contribution!

**Pull request merged**